crrExistingObjects.js

```diff
@@ -0,0 +1,166 @@
+const http = require('http');
+
+const AWS = require('aws-sdk');
+const { doWhilst, mapLimit, waterfall } = require('async');
+
+const { Logger } = require('werelogs');
+
+const BackbeatClient = require('./BackbeatClient');
+
+const log = new Logger('s3utils::emptyBucket');
+const BUCKETS = process.argv[2] ? process.argv[2].split(',') : null;
```

Comment:
I think it's fine for now, but I'm thinking the script will likely evolve a little bit, e.g. to support DR and reprocess REPLICA entries, and then it could be worth switching to a proper argument parsing module like "commander" for the extra options, actually even now it could be used e.g. to specify a different number of workers, as the default may not be a good fit for everyone. Using an argument-parsing module would also have the benefit of providing consistent help information for usage. Another way could be to pass everything as environment variables, but I don't think this would be the most usable approach.

Also it looks to me more natural to allow passing the buckets as a list of arguments instead of custom `split(',')` call, because most command-line tools already use this scheme to parse extra args (so to comply with principle of least surprise).